### PR TITLE
Rename Libbeat variables and improve Info output

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

**Comment by urso** [comment]
Refactor==rename?

**Comment by ruflin** [comment]
@urso title updated :-)

**Comment by urso** [comment]
LGTM

File: outputs/fileout/file.go [hunk 1]
```diff
@@ -38,22 +38,26 @@ func (out *fileOutput) init(beat string, config *outputs.MothershipConfig, topol
 	if out.rotator.Name == "" {
 		out.rotator.Name = beat
 	}
+	logp.Info("Logging Base Filename set to: %v", out.rotator.Name)
```
Comment (andrewkroh) [inline outputs/fileout/file.go]: "Logging"? This is "File Output".

Comment (ruflin) [inline outputs/fileout/file.go]: Fixed. Good you spotted this :-(

**Pull request merged**